flink-end-to-end-tests/flink-plugins-test/src/main/java/org/apache/flink/fs/dummy/DummyFileSystemFileStatus.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.fs.dummy;
+
+import org.apache.flink.core.fs.FileStatus;
+import org.apache.flink.core.fs.Path;
+
+class DummyFileSystemFileStatus implements FileStatus {
```

Comment:
`DummyFileStatus` would follow a pre-existing convention of `HadoopFileStatus` and `LocalFileStatus`

Reply:
If you insist in such change, I propose to revert back the renaming and change `DummyFileSystem` to `DummyFs` with keeping it as prefix in those classes:
 1) I'm personally neutral about using common abbreviations (id, cpu, url, db, fs and so on). And there are already implementations of `FileSystem` that use such convention;
 2) the prefix here is intended rather as a marker prefix, similar to "Hadoop", "FlinkS3", "MapR" in other `FileSystem` implementations. It doesn't mean that the corresponding `FileStatus/FSDataInputStream` implementations are dummy or mocks.

Reply:
Ok I get your point. In that case I would vote for one of the following:
1. `DummyFileSystem`, `DumyFileStatus`, `DummyDataInputStream`
2. `DummyFSFileSystem` `DumyFSFileStatus`, `DummyFSDataInputStream` (note `FS` which would also feet the code base better compared to `Fs` and note that all files should have the same prefix)
3. Rename it altogether, for example to `StaticContent` or `PredefinedContent` (`StaticContentFileSystem`, `StaticContentFileStatus`, `StaticContentDataInputStream`, ...)

I think the 3rd would be the best, as it also describes in what way the file system is "dummy", not just the fact that it is "dummy".

Reply:
I'm personally against option 1. As it's misleading that `DummyFileStatus`, `DummyDataInputStream`, ... are dummy, but actually they are valid implementations for corresponding FS.
I can adjust PR for option 2., by renaming `DummyFs` to `DummyFsFileSystem`. I was not aware about "abbreviation in uppercase" convention. In particular, I was using as examples of naming `HadoopFsFactory`, `MapRFsFactory`.

Initially, I also though about option 3, but `static` can't be used as a standalone part of package name. While `staticcontent` package name feels off style with other FSs.

Reply:
Ok, you are right about `HadoopFsFactory` and `MapRFsFactory`. I was looking mostly at different abbreviations. `DummyFsFileSystem` sounds good to me in that case.